test/unit/cli_test.rb

```diff
@@ -24,6 +24,7 @@ class CliTest < Minitest::Test
       offense = Offense.new(file: file_path, message: violation_message)
 
       configuration = Configuration.new({ "parallel" => false })
+      configuration.stubs(load_paths: [])
```

Comment:
should the load paths really be empty here?

Reply:
OK, yeah - I think it's correct for these tests. I developed doubts after seeing our [test fixtures](https://github.com/Shopify/packwerk/pull/150/files#diff-1f9d969781597a70faf79179a2fc6600c0aee1f329e32e390f7cff1c1c61b4b0) did have load paths, but I guess those aren't used here at all.